lib/src/disposable/disposable.dart

```diff
@@ -211,6 +252,13 @@ class Disposable implements _Disposable, DisposableManagerV3 {
   /// Dispose of the object, cleaning up to prevent memory leaks.
   @override
   Future<Null> dispose() async {
+    int startTime;
+    if (_debugMode) {
+      startTime = new DateTime.now().millisecondsSinceEpoch;
```

Comment:
could use a [`Stopwatch`](https://api.dartlang.org/stable/1.24.0/dart-core/Stopwatch-class.html) for this